.github/workflows/upload-installer-to-release.yaml

```diff
@@ -0,0 +1,29 @@
+#TODO: merge this workflow to the release-installer.yaml after the issue on arm hosts is fixed
+name: Upload installer
+on:
+  workflow_dispatch: # Trigger this workflow from tag
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: write   # This is required for uploading the release assets
+jobs:         
+  upload-installers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.ROLE }}
+          role-session-name: download-installer-session
+          aws-region: ${{ secrets.REGION }}
+      - name: Download installers
+        run: |
+          aws s3 cp s3://${{ secrets.INSTALLER_PRIVATE_BUCKET_NAME }}/Finch-${GITHUB_REF_NAME}-aarch64.pkg Finch-${GITHUB_REF_NAME}-aarch64.pkg
```

Comment:
What does `GITHUB_REF_NAME` evaluate to in this context? Is it the same as ${{ github.ref_name }} ?

Reply:
It's the same. [GITHUB_REF_NAME](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) is a default environment variable, and [github.ref_name](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) is the github context accessible via env vars as well. Their value is the tag that triggered this workflow.